sys/fido2/ctap/Kconfig

```diff
@@ -4,13 +4,31 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-menuconfig KCONFIG_USEMODULE_FIDO2_CTAP
+menuconfig MODULE_FIDO2_CTAP
     bool "FIDO2 CTAP"
-    depends on USEMODULE_FIDO2_CTAP
+    depends on HAS_PERIPH_GPIO
+    depends on HAS_PERIPH_GPIO_IRQ
+    depends on TEST_KCONFIG
+    select PACKAGE_TINYCBOR
+    select PACKAGE_MICRO-ECC
+    select PACKAGE_TINY-ASN1
+    select MODULE_PERIPH_GPIO
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_EVENT
+    select MODULE_EVENT_TIMEOUT
+    select MODULE_ZTIMER
+    select MODULE_ZTIMER_MSEC
+    select MODULE_MTD
+    select MODULE_MTD_FLASHPAGE
+    select MODULE_MTD_WRITE_PAGE
+    select MODULE_RANDOM
+    select MODULE_CRYPTO_AES_256
+    select MODULE_CIPHER_MODES
+    select MODULE_HASHES
     help
```

Comment:
Now we can change the `help` text to something more meaningful about the module. For starters we can just use the documentation from the ctap header file.

>  Say y to enable CTAP protocol support. The Client-to-Authenticator Protocol (CTAP) is an application layer protocol for the communication between an authenticator and a host.

Probably worth to mention FIDO as well, and some URI.